### ISSUE TITLE
Old sims do not load the a11y view from the PhET website

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/chipper#1580]
Found by @KatieWoe and discussed on slack: there are older sims with accessibility features where the a11y views do not load on the PhET website. The `src` attribute is incorrect for these sims and has brand info These sims were likely published before this line was added:

https://github.com/phetsims/chipper/blob/4162a56a93d1cc26b60658b13a963b5542eb12c5/wrappers/a11y-view/index.html#L252-L253

We will identify which sims need this fix and then patch them in a maintenance release.

**Comment by jessegreenberg** [comment]
I have too many assignments to get to this right now. For those sims where an a11y view is desired, you can find it on bayes. So this is a low priority task.

**Comment by jessegreenberg** [comment]
We should fix this now as part of https://github.com/phetsims/qa/issues/1386. I will follow up once the current batch is complete.

From the batch MR process, we will be able to easily identify the simulations that need this.

**Comment by jessegreenberg** [comment]
These are the sims that require a patch:

- [ ] [balloons-and-static-electricity 1.5.23 a11y view]
- [ ] [friction 1.6.21 a11y view]
- [ ] [gravity-force-lab-basics 1.1.25 a11y view]
- [ ] [greenhouse-effect 1.3.11 a11y view]
- [ ] [john-travoltage 1.6.24 a11y view]
- [ ] [mean-share-and-balance 1.1.7 a11y view]
- [ ] [my-solar-system 1.3.14 a11y view]
- [ ] [ratio-and-proportion 1.2.20 a11y view]
- [ ] [trig-tour 1.1.7 a11y view]

**Comment by jessegreenberg** [comment]
This is complete. Closing.
